<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поставщики</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_add_provider">
     <property name="text">
      <string>Добавить поставщика</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListWidget" name="lw_provider"/>
   </item>
   <item row="0" column="1" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_add_provider</tabstop>
  <tabstop>lw_provider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_add_provider</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>view_add_provider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lw_provider</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>select_provider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>view_add_provider()</slot>
  <slot>select_provider()</slot>
  <slot>exit_provider()</slot>
 </slots>
</ui>
